pandas/core/generic.py

```diff
@@ -7538,16 +7538,21 @@ def between_time(
         Parameters
         ----------
         start_time : datetime.time or str
+                The first time value.
```

Comment:
Too much indentation, should be four spaces.

Can you be more specific of what start_time should be? This description doesn't clarify much to me.

Reply:
Got it, let me try to add more detail.